src/ape_geth/provider.py

```diff
@@ -376,6 +383,10 @@ def get_call_tree(self, txn_hash: str) -> CallTreeNode:
         self._can_use_parity_traces = True
         return tree
 
+    def set_code(self, address: AddressType, code: ContractCode) -> bool:
+        # NOTE: Does no work on regular Geth nodes, will still raise APINotImpemented.
+        return self._make_request("debug_setCode", [address, code])
```

Comment:
Does this only work on geth dev?

Reply:
no it doesnt work anywhere, i just deleted